OpenTrons/opentrons-api
# feat(shared-data, labware-library): add thermo scientific labware definitions

re AUTH-699 and AUTH-700



# Overview



Add new Thermo Scientific labware definitions with correct module parameters and stacking offsets to shared data and Labware Library.

All new labware details can be found in [this spreadsheet](https://docs.google.com/spreadsheets/d/1odyozj6Ns-9g_jCI3DxArd3iP6WjKlBCAyoK1khR0rI/edit?usp=sharing).



## Test Plan and Hands on Testing


[New Labware Testing](https://docs.google.com/spreadsheets/d/1r7ldsZj9NxBprDMEges6QFphoLSJyfhqiwT7xg6-7f8/edit?usp=sharing) : This spreadsheet shows all the tests I completed on each of the Thermo Scientific labware definitions and other new labware definitions. It also includes the LPC offset data for each new labware when placed on a Temperature Module, Magnetic Block, Heater Shaker, and/or Thermocycler Module.

Testing on Flex:
- Measure stacking offset on selected module adapters using calipers and compare it with definitions.
- Run a protocol on a PVT robot with a PVT pipette to test the labware on compatible modules by going to the 4 corner wells.
- Run the Labware Position Check to test the labware dimensions (accepting up to +/- 0.5mm in both the x and y directions due to spring force on the deck slot).
- Test if the gripper can pick up the labware from any compatible module and place it in any slot without dropping or crashing.

Labware Library:

- Ensure that the new Thermo Scientific labware appears under the Well Plate section and that their labware images are displayed correctly : 

1. Thermo Scientific 96 Well Plate 200 µL
2. Thermo Scientific Nunc 384 Well Plate 40 µL
3. Thermo Scientific Abgene 96 Deep Well Plate 0.8mL
4. Thermo Scientific 96 Deep Well Plate 1.2mL
5. Thermo Scientific Nunc 96 Well Plate 450 µL




## Changelog



- Added Thermo Scientific JSON definitions to shared data
- Updated definitions with module parameters and stacking offsets
- Added Thermo Scientific labwares and their images to Labware Library and Labware section in Opentrons app 
- Added Thermo Scientific labwares to the lists in `QuickTransferFlow/constants.ts` file: `SINGLE_CHANNEL_COMPATIBLE_LABWARE`, `EIGHT_CHANNEL_COMPATIBLE_LABWARE`, and `NINETY_SIX_CHANNEL_COMPATIBLE_LABWARE`

## Review requests



## Risk assessment
